apps/src/code-studio/pd/components/schoolAutocompleteDropdownWithCustomFields.jsx

```diff
@@ -0,0 +1,134 @@
+/*
+ * Autocomplete school dropdown with
+ * custom fields if school not found
+ */
+import React, {PropTypes} from 'react';
+import {FormGroup, Row, Col, ControlLabel, HelpBlock} from 'react-bootstrap';
+import SchoolAutocompleteDropdown from '@cdo/apps/templates/SchoolAutocompleteDropdown';
+import CustomSchoolInfo from './customSchoolInfo';
+import {isZipCode} from '@cdo/apps/util/formatValidation';
+
+
+const VALIDATION_STATE_ERROR = "error";
+const OTHER_SCHOOL_VALUE = "-1";
+
+export default class SchoolAutocompleteDropdownWithCustomFields extends React.Component {
+  static propTypes = {
+    school_id: PropTypes.string,
+    school_info: PropTypes.object,
+    onSchoolInfoChange: PropTypes.func,
```

Comment:
Why do we have 2 input props: id and info, yet only one update callback? Do we need the `school_id` prop? Isn't that contained in the `school_info`?